src/main/java/by/exadel/internship/dto/enums/InterviewTime.java

```diff
@@ -0,0 +1,8 @@
+package by.exadel.internship.dto.enums;
+
+public enum InterviewTime {
```

Comment:
```suggestion
public enum UserInterviewDuration {
```

Reply:
I decided to delete this enam because we decided to use the int